### PR TITLE
set SSL ciphers

### PR DESCRIPTION
**Pull request by ghost**
Have added a way to disable/enable ssl ciphers.
Now sync uses openssl ciphers "HIGH:!DSS:!aNULL@STRENGTH" instead of weaker ciphers like RC4

### CODE REVIEW

**Comment by calzoneman** [comment]
From http://nodejs.org/api/tls.html#tls_tls_createserver_options_secureconnectionlistener:

> Note that it is still possible for a TLS v1.2 client to negotiate a weaker cipher unless `honorCipherOrder` is enabled.

So I think if you're going to change the cipher list you should probably set `honorCipherOrder: true` as well, no?

Getting rid of RC4 is fine by me, but are there any other ciphers your change drops that might cause browser/OS incompatibilities?

**Comment by ghost** [comment]
According to
https://www.ssllabs.com/ssltest/analyze.html?d=synchtu.be&latest the only
browsers this will break is internet explorer 6 on windows xp. I guess I
can look at setting honorCipherOrder: true tomorrow

File: lib/server.js [hunk 1]
```diff
@@ -72,7 +72,8 @@ var Server = function () {
         key: key,
         cert: cert,
         passphrase: Config.get("https.passphrase"),
-        ca: ca
+        ca: ca,
+	ciphers: Config.get("https.ciphers")
```
Comment (calzoneman) [inline lib/server.js]: Please fix indentation

**Comment by calzoneman** [comment]
As long as you've looked into it and determined that what is in config.template.yaml is a sensible default, I'm good to merge this.

**Pull request merged**